trend-analysis/TrendAnalysisApp.UnitTests/TrendDataProcessorTests.cs

```diff
@@ -45,14 +81,258 @@ public void TestUpdateIndicatorSearchDataWritesCorrectDataToFile() {
         mockIndicatorDataForSearch.AreaToTrendList.Add(
             new AreaWithTrendData("ABCAreaCode", "Increasing and getting worse")
         );
-        var mockPerIndicatorData = new List<IndicatorTrendDataForSearch> {mockIndicatorDataForSearch};
+        var mockPerIndicatorData = new List<IndicatorTrendDataForSearch> { mockIndicatorDataForSearch };
 
         _trendDataProcessor.UpdateIndicatorSearchData(mockPerIndicatorData);
 
-        _mockFileHelper.Received().Read(expectedFilePath);
+        _mockFileHelper.Received().Read(ExpectedFilePath);
         _mockFileHelper.Received().Write(
-            expectedFilePath,
+            ExpectedFilePath,
             Arg.Is<JArray>(ja => JToken.DeepEquals(ja, expectedUpdatedFileContents))
         );
     }
+
+    [Fact]
+    public async Task TestTrendDataProcessor_DoesNotReturnTrendsForIndicatorsToSkip()
+    {
+        // arrange
+        JArray mockFileContents = new()
+        {
+            { new JObject { ["indicatorID"] = 1 } }, { new JObject { ["indicatorID"] = 2 } } 
```

Comment:
```suggestion
            { new JObject { ["indicatorID"] = 1 } }, { new JObject { ["indicatorID"] = 6 } } 
```
Or could just have one object. Does not really matter in this test case.